components/SVGWebView.tsx

```diff
@@ -259,3 +239,6 @@ var styles = StyleSheet.create({
     fontSize: 14
   }
 });
+
+// 2do: Make ShopList button look pretty in iPhoneX

```

Comment:
Did you not do this with the safe area view?

Reply:
Yes, the button now works, but it shows 'ugly' in case of iPhoneX.
But I left this for a future iteration.

Reply:
I will also now change the button from 'contained' to 'text' for iOS only. Not a perfect fix, but still a bit prettier.